src/decision/Criteria.jl

```diff
@@ -44,6 +44,15 @@ Base.@kwdef struct Criteria{P,N} <: EcoModel
         name="Outgoing Connectivity (Seed)",
         description="Higher values give preference to locations with high outgoing connectivity (i.e., provides larvae to other sites) for enhanced coral deployments.",
     )
+    seed_depth::P = Param(
```

Comment:
Everywhere else you are calling this `depth` so, for consistency, I would change to `depth` here as well.

Also, a question: this is not the seed depth, but the location depth, right? (otherwise maybe everywhere else should change to `seed_depth` instead..)

Reply:
For the other weights if they are not used for multiple interventions, we tend to have the intervention it is used for in the name. This is not completely consistent across parameters but I intend to fix this in #559

Reply:
But then why not changing `wt_depth`  to `wt_seed_depth`?

Reply:
I can do that :)